src/bindings/xfconf/README.md

```diff
@@ -0,0 +1,103 @@
+- infos =
+- infos/author = Richard Stöckl <e11908080@student.tuwien.ac.at>
+- infos/licence = BSD
+- infos/status = experimental maintained
+- infos/provides =
+- infos/description = Elektra Bindings for xfconf
+
+## Introduction
+
+These bindings provide an implementation of the xfconf header files such that libelektra can be used as a drop-in replacement for xfconf.
+In other words, it makes applications which rely on xfconf for their configuration use libelektra without the necessity for modifying their source code or even recompiling them.
+
+For further understanding how xfconf is structured, please refer to the [xfconf plugin](../../../src/plugins/xfconf/README.md#xfconf-terminology).
```

Comment:
As the binding is probably the more important artifact, I would prefer if here is the general information and the plugin links here.

Reply:
Should be done by 6506d0e23efa25047381ef0cd2afa5aed13da3f7 or before